benches/elf_loader.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Solana <alexander@solana.com>
```

Comment:
Credit Solana maintainers

Reply:
You mean `Solana Maintainers <maintainers@solana.com>`?